src/redux/actions/academicsActions.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+// var url = '/dummy/json/academics.json';
+// var url = '/dummy/json/academics_reserved_seats.json';
+
+export const FETCH_ACADEMICS_START = 'FETCH_ACADEMICS_START';
+export const FETCH_ACADEMICS_SUCCESS = 'FETCH_ACADEMICS_SUCCESS';
+export const FETCH_ACADEMICS_FAILURE = 'FETCH_ACADEMICS_FAILURE';
+
+export const fetchAcademicsStart = () => ({
+  type: FETCH_ACADEMICS_START
+});
+
+export const fetchAcademicsSuccess = academics => ({
+  type: FETCH_ACADEMICS_SUCCESS,
+  value: academics
+});
+
+export const fetchAcademicsFailure = error => ({
+  type: FETCH_ACADEMICS_FAILURE,
+  value: error
+});
```

Comment:
@developish - Why are we defining the failure, if it's not used here. Is this something that will be imported elsewhere to handle failure of the request? Is this just boiler plate for when/if we need to use it to process failure?

Reply:
Updated to include error handling into the Redux store